x/lockup/keeper/lock_test.go

```diff
@@ -144,47 +144,17 @@ func (suite *KeeperTestSuite) TestUnlockPeriodLockByID() {
 	suite.Require().Len(locks, 0)
 }
 
-func (suite *KeeperTestSuite) TestLock() {
```

Comment:
Does it make sense to do either or both:
-  test `CreateLock`
- use the [`export_test.go`](https://medium.com/@robiplus/golang-trick-export-for-test-aa16cbd7b8cd) trick that will allow us to
    * export `lock` for tests only and
    * keep testing the now unexported `lock` method

Reply:
Ah missed the test for `CreateLock` thanks for catching this! Just added this in the most recent commit!